client/src/+assets/assets-list.module.scss

```diff
@@ -243,7 +243,7 @@
       border-color: gc(gM);
       border-style: solid;
       vertical-align: middle;
-      padding: 5px 15px;
+      padding: 18px 15px;
```

Comment:
Quick fix of the min height in asset list.

The problem:
<img width="835" alt="Screenshot 2022-09-26 at 16 24 25" src="https://user-images.githubusercontent.com/1643399/192317103-7920d43c-41ec-456b-b7e0-7bc9ca35b574.png">

The fix:
<img width="835" alt="Screenshot 2022-09-26 at 16 24 12" src="https://user-images.githubusercontent.com/1643399/192317167-206a2bfa-6f8f-4b90-b8e5-fb11c3071348.png">